cypress/e2e/pages/articles/testsForLiteOnly.js

```diff
@@ -1,34 +1,30 @@
-import { liteEnabledServices } from '#app/components/LiteSiteCta/liteSiteConfig';
-
 export default ({ service, pageType }) => {
-  if (liteEnabledServices.includes(service)) {
-    describe(`Running testsForLiteOnly for ${service} ${pageType}`, () => {
-      describe('CTA: Lite', () => {
-        it('Clicking the link to the main site should navigate to canonical site', () => {
-          cy.get('[data-e2e="to-main-site"]').within(() => {
-            cy.get('a')
-              .should('have.attr', 'href')
-              .then($href => {
-                cy.get('a').click();
-                cy.url().should('eq', $href).should('not.contain', '.lite');
-              });
-          });
-          cy.go('back');
+  describe(`Running testsForLiteOnly for ${service} ${pageType}`, () => {
+    describe('CTA: Lite', () => {
```

Comment:
Can we rename this - I'd like to get away from Lite Site CTA as much as we can, and instead use the Lite Site Summary terminology

```suggestion
    describe('Lite Site Summary', () => {
```